decidim_app-design/app/views/public/partials/_component_icons.html.erb

```diff
@@ -1,48 +1,16 @@
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "information" %>
-    <span>Pàgina</span>
-  </div>
-</div>
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "process" %>
-    <span>Procés</span>
-  </div>
-</div>
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "meetings" %>
-    <span>Trobades</span>
-  </div>
-</div>
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "proposals" %>
-    <span>Propostes</span>
-  </div>
-</div>
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "debates" %>
-    <span>Debats</span>
-  </div>
-</div>
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "actions" %>
-    <span>Resultats</span>
-  </div>
-</div>
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "polls" %>
-    <span>Enquestes</span>
-  </div>
-</div>
-<div class="m-bottom">
-  <div class="process-nav__link">
-    <%= icon "budgets" %>
-    <span>Pressupostos</span>
-  </div>
+<%
+file = Rails.root.join("app/assets/images/icons.svg")
+doc = File.open(file) { |f| Nokogiri::XML(f) }
+titles = doc.css("title").map(&:text)
```

Comment:
Yeeeeeeah! Nice job. Please kill my hacky "showcase" view and my hacky spec (`spec/icons_spec.rb`)

Reply:
Done!